### ISSUE TITLE
[Reports] Disable "Header row" option in .csv exports of packing reports

### ISSUE THREAD

**Issue by audez** [openfoodfoundation/openfoodnetwork#9546]
## Description
In Reports > Packing reports(admin/reports/packing), the display option "header row" should be disabled for .csv exports.

## Expected Behavior
In the packing reports the "header row" checkbox should be greyed out, as it is not possible to display the header row in csv files (identically to the "summary row", see #9291 ).

## Actual Behaviour
The "header row" checkbox can be checked, but this has no effect on the export file.

## Steps to Reproduce
1. Connect to an entreprise's admin (hub, shop or producer) that has at least one order
2. Go to Reports > Packing reports (/admin/reports/packing)
3.  In Rendering Options: check the "header row" box
6. In "generate report", choose csv, click "GO" 

## Animated Gif/Screenshot
![Screen Shot 2022-08-09 at 21 43 06](https://user-images.githubusercontent.com/28386648/183747317-279a7b27-a24f-4ff8-beab-bbcfcdd4ff24.png)

## Severity
bug-s3: a feature is broken but there is a workaround

**Comment by jibees** [comment]
Sorry, but I think it's a duplicate from #9291 (which should be edited I think)

**Comment by RachL** [comment]
@jibees as there is already a PR attached, I was too lazy to update it :)

**Comment by RachL** [comment]
> This was apparently working well before a recent release, but I can't find the related issue.

@audez FYI header row were never included in .csv. Total rows were, perhaps they were referring to that? In any case, I'm surprised they like .csv : do you know why they use this format? We are not going to add any formatting to .csv form anymore, it will stay a "raw" version. The .xlsx version is the one with the shiny stuff :D So I'm wondering what's blocking them to use the spreadsheet version? 🤔

**Comment by audez** [comment]
Thanks guys! And sorry for the duplicate, my issue searching skills really needs improvement 😳
I thought the total rows were there in the csv, so i figured he was talking about the summary rows. But yes you're right, as I see now, there's no total row in the csv neither, and that was the problem. I just called him and he said he used csv because he didn't see the others, but now it's fine he will use the shiny Excel ✨

**Comment by RachL** [comment]
cooool :) Nice one @audez :muscle: And again don't worry about duplicates ;)

**Comment by filipefurtad0** [comment]
I think we might need to re-open this one as https://github.com/openfoodfoundation/openfoodnetwork/pull/9563 only affects the summary row, [and not the header row](https://github.com/openfoodfoundation/openfoodnetwork/pull/9563#issuecomment-1218190926)?

**Comment by audez** [comment]
Yeah actually the header raw was only mentioned in the comments in #9291 . As far as I understand, both checkboxes should be disabled as CSV is just a raw version. Do you think I could change this issue to "grey out the header row option", and reopen it, or is it better to create a new issue? 
Too bad as it could have been done at once 😬

**Comment by filipefurtad0** [comment]
Thanks @audez, I'd say updating and reopening would be fine :+1:
